src/workerd/api/crypto/dh.c++

```diff
@@ -100,6 +103,10 @@ kj::Own<DH> initDh(kj::OneOf<kj::Array<kj::byte>, int>& sizeOrKey,
             }
             return 1;
           };
+          // Operations on an "egregiously large" prime will throw with recent boringssl.
```

Comment:
I assume later we'd be (hopefully) updating these to throws? If so, a `TODO` comment indicating next steps would be good on these.

Reply:
Yup, I had a TODO message in a previous version of this. Added one here again.